src/Keyboard/Keyboard.php

```diff
@@ -147,21 +152,23 @@ public static function inlineButton($params = []): string
      *
      * <code>
      * $params = [
-     *   'hide_keyboard' => true,
+     *   'remove_keyboard' => true,
      *   'selective'     => false,
      * ];
      * </code>
      *
-     * @link https://core.telegram.org/bots/api#replykeyboardhide
+     * @link https://core.telegram.org/bots/api#replykeyboardremove
      *
-     * @param array $params
+     * @param array $params          [
      *
-     * @var bool    $params ['hide_keyboard']
-     * @var bool    $params ['selective']
+     * @type bool   $remove_keyboard Required. Requests clients to remove the custom keyboard (user will not be able to summon this keyboard; if you want to hide the keyboard from sight but keep it accessible, use one_time_keyboard in ReplyKeyboardMarkup)
+     * @type bool   $selective       Optional. Use this parameter if you want to remove the keyboard for specific users only. Targets: 1) users that are @mentioned in the text of the Message object; 2) if the bot's message is a reply (has reply_to_message_id), sender of the original message.
+     *
+     * ]
      *
      * @return string
      */
-    public static function hide(array $params = []): string
+    public static function remove(array $params = []): string
```

Comment:
Return would be `Keyboard` if it's keyboard for the next method too. See `forceReply`.